docker-compose-ptvsd.yaml

```diff
@@ -0,0 +1,7 @@
+version: '3'
```

Comment:
Any consideration to running this from a directory instead? I think this is a cool idea, but it would be nice not to add yet another file to the root level.

Reply:
I'm not sure what you mean. We could probably move all of the build files (Dockerfile, docker-compose*) into a subdirectory if you wanted to clean up the root.

Reply:
I mean, yeah, that's exactly what I mean. I feel there might be some value in having `Dockerfile` and `docker-compose.yaml` in the root for transparency (but maybe I could be convinced otherwise), but having this extra `docker-compose-ptvsd.yaml` only for use with specific technology at the root level ... I dunno. I just wanted to know if you'd considered this, I'm not asking for the change. Could it go in `.vscode`?

Reply:
Sure this file could go anywhere, it just needs to be on the command line. I feel like the biggest negative of a ton of files on the root is when viewing on Github you have to scroll to see the README like with [Open Edx](https://github.com/edx/edx-platform) or [Canvas](https://github.com/instructure/canvas-lms). Wow Canvas has like 10 docker-compose* files on the root! :)

Reply:
I feel like this thread is bikeshedding as @johnpj would say. ;)

Reply:
Yeah, it also bugs me in VSCode, since I like to right click on open space in the left-pane directory tree to create a new directory, and when there are a million root-level files (MyLA comes to mind), there isn't much space to work with.

Reply:
I mean, fine, I'll take the put down in stride. I like things to be organized, that's just part of who I am.

Reply:
I wouldn't consider it a "put down", I just think if we want to clean up the root it should be on a different issue rather than one-off. At least half of the files could be moved. I mostly use the terminal is VSCode to do a lot of stuff so I've never right clicked in the left pane, but different people work differently!

Reply:
Created at #171. I'll move this file for now to end this thread and move on to another issue.